native/android/app/build.gradle

```diff
@@ -226,14 +226,19 @@ android {
             }
 
         }
-    }
 
-    // Exclude mapbox-android-accounts because of its proprietary license
-    applicationVariants.all { variant ->
+        // Exclude mapbox-android-accounts because of its proprietary license
         variant.getCompileConfiguration().exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-accounts'
         variant.getRuntimeConfiguration().exclude group: 'com.mapbox.mapboxsdk', module: 'mapbox-android-accounts'
     }
 
+    // Exclude proprietary firebase for floss builds
+    if (buildConfig.featureFlags.floss) {
+        configurations {
+            all*.exclude group: 'com.google.firebase'
+        }
+    }
```

Comment:
Does this include runtime and compile configurations? Did you check after building whether the class fields of firebase are actually excluded?

Reply:
We probably also need to exclude `com.google.android.gms`. It brings for example `play-services-cloud-messaging`.

If we exclude whole GMS right away we possible break our location code. I'm not sure how to proceed here.

Reply:
It should, yes. I checked with https://github.com/Exodus-Privacy/exodus-standalone and it says firebase is no longer included (also no google play services like `com.google.android.gms` or similar). How would I check this manually? But I guess we don't need to exclude anything else (at least for firebase).

Reply:
That should be fine then. Did you check a release or debug build?
I used the decompiler of intellij previously. If you see something in kursiv, then it is not actually included.

Reply:
I checked the release build. Ok, will try, thx.